src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHandlerManager.cs

```diff
@@ -44,16 +44,31 @@ public void Handle(IProjectVersionedValue<IProjectSubscriptionUpdate> update, Ru
             Requires.NotNull(update, nameof(update));
             Requires.NotNull(context, nameof(context));
 
-            if (handlerType == RuleHandlerType.Evaluation)
+            dynamic dynamicContext = context;
```

Comment:
Why the dynamic, given you're not using it in the end and using classic reflection?